rob1_hand_over_pin = [1.2611360549926758, -1.4101837438395997, 0.8596294561969202, -2.5924040279784144, -1.26237661043276, 1.574709415435791]
rob1_pre_hand_over_pin = [1.1898112297058105, -1.6090036831297816, 1.0736168066607874, -2.6081143818297328, -1.190704647694723, 1.5754528045654297]

rob2_hand_over_pin = [1.229053020477295, -1.3147557538798829, 0.7326763311969202, -2.559657712975973, -1.2300236860858362, -1.5678418318377894]
rob2_pre_hand_over_pin = [1.1643433570861816, -1.51138527811084, 0.9686983267413538, -2.5994135342040003, -1.1651447455035608, -1.567374054585592]

rob1_pin1_pre_grasp = [2.0929408073425293, -1.6921888790526332, 2.246387783681051, -2.1255299053587855, -1.5662253538714808, 2.096367359161377]
rob1_pin1_grasp = [2.093252658843994, -1.5034073156169434, 2.3713038603412073, -2.4392763576903285, -1.5671966711627405, 2.097517490386963]

rob1_home = [-1.5708072821246546, -0.785393552189209, -1.570775032043457, -1.5707853597453614, 1.5708041191101074, 6.389617919921875e-05]
rob2_home = [-1.5708006064044397, -0.7853620809367676, -1.570786476135254, -1.57081999401235, 1.5708045959472656, 6.198883056640625e-06]

rob1_lpin1_grasp = [-2.267552200947897, -1.5506490033916016, -2.410569190979004, -0.7509868902019043, 1.5700364112854004, 0.8775439262390137]
rob1_lpin2_grasp = [-2.342891518269674, -1.6326991520323695, -2.3256635665893555, -0.7538145345500489, 1.5698208808898926, 0.8019795417785645]
rob1_lpin3_grasp = [-2.409832779561178, -1.7132541141905726, -2.2320985794067383, -0.7668274205974122, 1.569653034210205, 0.7348642349243164]
rob1_lpin4_grasp = [-2.4688003698932093, -1.7927071056761683, -2.130356788635254, -0.7891262334636231, 1.5695204734802246, 0.6756548881530762]

rob1_lpin1_pre_grasp = [-2.2611377874957483, -1.3809469503215333, -2.2994165420532227, -1.0317552846721192, 1.5707926750183105, 0.8837461471557617]
rob1_lpin2_pre_grasp = [-2.3382566610919397, -1.4773214620402833, -2.224909782409668, -1.009927586918213, 1.5705046653747559, 0.8064994812011719]
rob1_lpin3_pre_grasp = [-2.410480324422018, -1.5781403980650843, -2.1348772048950195, -0.9991820615581055, 1.5702166557312012, 0.7341446876525879]
rob1_lpin4_pre_grasp = [-2.468741242085592, -1.6565619907774867, -2.0291337966918945, -1.0264786046794434, 1.5700125694274902, 0.6756901741027832]

rob1_longpart_hole1 = [-1.9427803198443812, -2.11822046856069, -1.4069147109985352, -1.1870072645, 1.569352626800537, 2.7711682319641113]
rob1_longpart_hole2 = [-1.592245880757467, -1.9162575207152308, -1.6686296463012695, -1.1272056859782715, 1.5695805549621582, 3.122347354888916]

rob1_shortpart_hole1 = [-1.8195694128619593, -2.380068918267721, -0.9348993301391602, -1.3971250814250489, 1.5691370964050293, 2.8931431770324707]
rob1_shortpart_hole2 = [-1.6155479590045374, -2.2446352444090785, -1.1662492752075195, -1.3012350362590333, 1.5692448616027832, 3.0977606773376465] 

rob1_leftchairpart_hole1 = [-1.288168732319967, -1.6939145527281703, -1.9200506210327148, -1.098048524265625, 1.5700125694274902, 1.9026284217834473]
rob1_leftchairpart_hole2 = [-1.2241981665240687, -1.6841846905150355, -1.8942804336547852, -1.1336115163615723, 1.5700840950012207, 1.9664673805236816]

rob1_leftchairpart_hold1_grasp = [-0.9880602995501917, -2.002401968041891, -1.9145746231079102, -0.8480819028666993, 1.604234218597412, 2.201660633087158]
rob1_leftchairpart_hold1_pre_grasp = [-0.9876883665667933, -1.7988087139525355, -1.7389936447143555, -1.2271879476359864, 1.6049551963806152, 2.201829433441162]

rob2_longpart_hole1_pre_grasp = [-1.234347168599264, -2.489133497277731, -0.686161994934082, -1.5380237859538575, 1.5716209411621094, 1.9065899848937988]
rob2_longpart_hole1_grasp = [-1.234610382710592, -2.5707441769041957, -0.8929834365844727, -1.2495811742595215, 1.5716328620910645, 1.9069256782531738]

rob2_longpart_hole2_grasp = [-1.0660160223590296, -2.7088433704771937, -0.6193876266479492, -1.3850506109050293, 1.5717287063598633, 2.074796199798584]
rob2_longpart_hole2_pre_grasp = [-1.0658357779132288, -2.6742121181883753, -0.4239816665649414, -1.615122457543844, 1.5716447830200195, 2.074545383453369]

rob2_shortpart_hole2_grasp = [-0.9758551756488245, -2.352065702477926, -1.3125648498535156, -1.0485952657512208, 1.5714287757873535, 2.1667189598083496]
rob2_shortpart_hole2_pre_grasp = [-0.9757598082171839, -2.2482491932311, -1.2101612091064453, -1.2548216146281739, 1.5715370178222656, 2.166527271270752]

rob2_shortpart_hole1_grasp = [-1.1842296759234827, -2.245986124078268, -1.5194673538208008, -0.9478867811015625, 1.571380615234375, 1.9588608741760254]
rob2_shortpart_hole1_pre_grasp = [-1.184134308491842, -2.106399198571676, -1.3948125839233398, -1.2121170324138184, 1.5716447830200195, 1.9586691856384277]

rob2_leftchairpart_hold1_grasp = [-1.4382885138141077, -2.2117654285826625, -1.5741510391235352, -0.9275387090495606, 1.5713567733764648, 0.1346592903137207]
rob2_leftchairpart_hold1_pre_grasp = [-1.4383123556720179, -2.061367174188131, -1.4368305206298828, -1.215240792637207, 1.571669101715088, 0.13446760177612305]


# <group_state name="rob2_pre_hand_over_pin" group="rob2_arm">
#     <joint name="rob2_shoulder_pan_joint" value="1.1643433570861816" />
#     <joint name="rob2_shoulder_lift_joint" value="-1.51138527811084" />
#     <joint name="rob2_elbow_joint" value="0.9686983267413538" />
#     <joint name="rob2_wrist_1_joint" value="-2.5994135342040003" />
#     <joint name="rob2_wrist_2_joint" value="-1.1651447455035608" />
#     <joint name="rob2_wrist_3_joint" value="-1.567374054585592" />
# </group_state>

srdf = "<group_state name=\"+name+\" group=